en/docs/how-tos/generating-backend-jwt.md

```diff
@@ -0,0 +1,120 @@
+# Generating Backend JWT 
+
+You can use JWT Generation in API Microgateway to send a customized JWT to the backend with user preferred claims. You have three possible options when selecting a method to send a JWT to the backend.
+
+- [Passing a token to the backend](#passing-a-token-to-the-backend)
+- [Generating a JWT inside Microgateway](#generating-a-jwt-inside-microgateway)
+
+### Passing a token to the backend
+
+When you are using JWT authentication, you can pass an already generated JWT token in the 'backendJwt' claim in the authentication JWT token. Therefore, if the authentication token consist the 'backendJwt' claim in the payload, the token will be set on to the desired header when sending to the backend without a generation process.
+
+####Sample payload
+
+```json
+{
+    "sub": "admin@carbon.super",
+    "aud": "JzCLuJq6kHIpMhK7C5OhiWN7cPsa",
+    "nbf": 1595447942,
+    "azp": "JzCLuJq6kHIpMhK7C5OhiWN7cPsa",
+    "scope": "am_application_scope default",
+    "iss": "https://localhost:9443/oauth2/token",
+    "exp": 1595451542,
+    "iat": 1595447942,
+    "jti": "0987c6d3-f11a-402c-8eb2-b21fb7b97f4d",
+    "backendJwt": "eyJ4NXQiOiJNell4TW1Ga09HWXdNV0kwWldObU5EY3hOR1l3WW1NNFpUQTNNV0kyTkRBelpHUXpOR00wWkdSbE5qSmtPREZrWkRSaU9URmtNV0ZoTXpVMlpHVmxOZyIsImtpZCI6Ik16WXhNbUZrT0dZd01XSTBaV05tTkRjeE5HWXdZbU00WlRBM01XSTJOREF6WkdRek5HTTBaR1JsTmpKa09ERmtaRFJpT1RGa01XRmhNelUyWkdWbE5nX1JTMjU2IiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiJhZG1pbkBjYXJib24uc3VwZXIiLCJhdWQiOiJKekNMdUpxNmtISXBNaEs3QzVPaGlXTjdjUHNhIiwibmJmIjoxNTk1NDQ3OTQyLCJhenAiOiJKekNMdUpxNmtISXBNaEs3QzVPaGlXTjdjUHNhIiwic2NvcGUiOiJhbV9hcHBsaWNhdGlvbl9zY29wZSBkZWZhdWx0IiwiaXNzIjoiaHR0cHM6XC9cL2xvY2FsaG9zdDo5NDQzXC9vYXV0aDJcL3Rva2VuIiwiZXhwIjoxNTk1NDUxNTQyLCJpYXQiOjE1OTU0NDc5NDIsImp0aSI6IjA5ODdjNmQzLWYxMWEtNDAyYy04ZWIyLWIyMWZiN2I5N2Y0ZCJ9.Ewz50wbRkFnod4-l5iQzuayHG6O5820Ykg8fQcj_7mIBDAOnWCk7j2a9KOFlJtMyqUXfkchgZEOM0sN9o7GmyYPxpw7x0pjHalNlrdNX8uwnrwaRfgn7G_9d57iFEUTyo7RHfR8uYNJYzSjHgd4x-Vy0-8fZm2OErtoVA0gH7pwnhnE8x3AQOTnBKbGEsWRoR6UmRNGCDwr23GlQ5v7GbPj4puJiE5Z-p5DNOtWuWX1BOpn9IpNn3nio-9wvE9DQUssDojn2G6t8BlM9vv9gMJJT3zK64tfCoWuhTMPn74vs3zGuYy2KO7x7dtULcv2o56slFCqHAID44KuB93vS7A"
+}
+```
+
+After altering the configuration, you can run API Microgateway. When you are invoking a resource with a JWT token, you will be able to obtain the generated backend JWT.
```

Comment:
Altering which configuration?

Reply:
I will section to the bottom with some improvements, like a conclusion to the doc.